sdc/hiframes/pd_series_ext.py

```diff
@@ -494,7 +494,7 @@ def resolve_value_counts(self, ary, args, kws):
 
 #     return typer
 
-str2str_methods = ['capitalize', 'swapcase', 'title']
+# str2str_methods = ['capitalize', 'swapcase', 'title']
```

Comment:
Please check all the places where the variable is used to make sure that this change doesn't break something. I have suspicions that it can break smth.